### ISSUE TITLE
Problem installing mongo extension

### ISSUE THREAD

**Issue by core23** [shivammathur/setup-php#135]
**Describe the bug**
When I want to run composer install using this action, the mongo extension seems missing.

**To Reproduce**
You can find the execution here: 
https://github.com/core23/setup-php-mongo-bug/commit/25a4ba2d3b0f1ad5d027c2ab6700f2590b4fafd4/checks?check_suite_id=380472442

```yaml
on:
  pull_request:
  push:
    branches:
      - master
    tags:
      - "**"

name: "Continuous Integration"

jobs:
  test-mongo:
    name: "Test Mongo"

    runs-on: ubuntu-latest

    strategy:
      matrix:
        php-version:
          - 7.4

    steps:
      - name: "Checkout"
        uses: actions/checkout@v1

      - name: "Install PHP with extensions"
        uses: shivammathur/setup-php@v1
        with:
          coverage: none
          extensions: "mbstring, json, mongo"
          php-version: ${{ matrix.php-version }}

      - name: "Install locked dependencies with composer"
        run: composer install --no-interaction --no-progress --no-suggest

```

**Expected behavior**
When adding the mongo extensions, composer will find it.

**Comment by shivammathur** [comment]
@core23 `ext-mongo` is not available for PHP 7.0 or newer. It only supports PHP 5.6.
To use `doctrine/mongodb` with PHP 7.0 or newer use `ext-mongodb` with `alcaeus/mongo-php-adapter`

See here for more info
- https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/1.2/reference/introduction.html#using-php-7

Try 
```yml
on:
  pull_request:
  push:
    branches:
      - master
    tags:
      - "**"

name: "Continuous Integration"

jobs:
  test-mongo:
    name: "Test Mongo"

    runs-on: ubuntu-latest

    strategy:
      matrix:
        php-version:
          - 7.4

    steps:
      - name: "Checkout"
        uses: actions/checkout@v1

      - name: "Install PHP with extensions"
        uses: shivammathur/setup-php@v1
        with:
          coverage: none
          extensions: "mbstring, json, mongodb"
          php-version: ${{ matrix.php-version }}

      - name: "Install locked dependencies with composer"
        run: |
          composer config "platform.ext-mongo" "1.6.16" && composer require "alcaeus/mongo-php-adapter"
          composer install --no-interaction --no-progress --no-suggest
```

**Comment by core23** [comment]
Thanks for the fast reply @shivammathur :)
